_rules/image-accessible-name-descriptive-qt1vmo.md

```diff
@@ -0,0 +1,295 @@
+
+This `svg` element has an `aria-label` attribute that describes the image.
+
+```html
+<html lang="en">
+	<svg viewBox="0 0 512 512" aria-label="HTML 5 logo">
+		<path
+			d="M108.4 0h23v22.8h21.2V0h23v69h-23V46h-21v23h-23.2M206 23h-20.3V0h63.7v23H229v46h-23M259.5 0h24.1l14.8 24.3L313.2 0h24.1v69h-23V34.8l-16.1 24.8l-16.1-24.8v34.2h-22.6M348.7 0h23v46.2h32.6V69h-55.6"
+		/>
+		<path fill="#e44d26" d="M107.6 471l-33-370.4h362.8l-33 370.2L255.7 512" />
+		<path fill="#f16529" d="M256 480.5V131H404.3L376 447" />
+		<path
+			fill="#ebebeb"
+			d="M142 176.3h114v45.4h-64.2l4.2 46.5h60v45.3H154.4M156.4 336.3H202l3.2 36.3 50.8 13.6v47.4l-93.2-26"
+		/>
+		<path fill="#fff" d="M369.6 176.3H255.8v45.4h109.6M361.3 268.2H255.8v45.4h56l-5.3 59-50.7 13.6v47.2l93-25.8" />
+	</svg>
+</html>
+```
+
+#### Passed Example 3
+
+This `canvas` element has an `aria-label` attribute that describes the image.
+
+```html
+<html lang="en">
+	<canvas id="act" width="200" height="60" aria-label="ACT Rules!"></canvas>
+	<script>
+		const ctx = document.querySelector('#act').getContext('2d')
+		ctx.font = '30px Arial'
+		ctx.fillText('ACT Rules!', 20, 40)
+	</script>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `img` element has an `alt` attribute that incorrectly describe the image.
+
+```html
+<html lang="en">
+	<img src="/test-assets/shared/w3c-logo.png" alt="ERCIM" />
+</html>
+```
+
+#### Failed Example 2
+
+This `svg` element has an `aria-label` attribute that incorrectly describe the image.
+
+```html
+<html lang="en">
+	<svg viewBox="0 0 512 512" aria-label="W3C logo">
+		<path
+			d="M108.4 0h23v22.8h21.2V0h23v69h-23V46h-21v23h-23.2M206 23h-20.3V0h63.7v23H229v46h-23M259.5 0h24.1l14.8 24.3L313.2 0h24.1v69h-23V34.8l-16.1 24.8l-16.1-24.8v34.2h-22.6M348.7 0h23v46.2h32.6V69h-55.6"
+		/>
+		<path fill="#e44d26" d="M107.6 471l-33-370.4h362.8l-33 370.2L255.7 512" />
+		<path fill="#f16529" d="M256 480.5V131H404.3L376 447" />
+		<path
+			fill="#ebebeb"
+			d="M142 176.3h114v45.4h-64.2l4.2 46.5h60v45.3H154.4M156.4 336.3H202l3.2 36.3 50.8 13.6v47.4l-93.2-26"
+		/>
+		<path fill="#fff" d="M369.6 176.3H255.8v45.4h109.6M361.3 268.2H255.8v45.4h56l-5.3 59-50.7 13.6v47.2l93-25.8" />
+	</svg>
+</html>
+```
+
+#### Failed Example 3
+
+This `canvas` element has an `aria-label` attribute that incorrectly describe the image.
+
+```html
+<html lang="en">
+	<canvas id="act" width="200" height="60" aria-label="HTML 5 logo"></canvas>
+	<script>
+		const ctx = document.querySelector('#act').getContext('2d')
+		ctx.font = '30px Arial'
+		ctx.fillText('ACT Rules!', 20, 40)
+	</script>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+This `img` element has an empty (`""`) [accessible name][]. The image is described by an adjacent text.
+
+```html
+<img src="/test-assets/shared/pdf-icon.png" alt="" /> PDF document
+```
+
+#### Inapplicable Example 2
+
+This `img` element has an empty (`""`) [accessible name][] because of its `presentation` role.
```

Comment:
Technically, "because of its `presentation` role" is incorrect in that case. Whether `role="presentation"` is here or not, the image has an empty alt text.
Then  do agree that we want to keep `role="presentation"` in the example in order to have it pass the SC…
What about
```suggestion
This decorative `img` element has an empty (`""`) [accessible name][] because it has no attributes or content to provide an accessible name.
```